locale/de/LC_MESSAGES/django.po

```diff
@@ -43,30 +43,45 @@ msgid ""
 "    <p>Here's a rough outline of what we're trying to build:\n"
 "    "
 msgstr ""
+"\n"
+"    <p>Wir sind ein paar Typen aus der Schweiz mit der Idee ein System zu "
+"entwickeln um dem allgemeine \\\"Drohnen Hass\\\" und der übertriebenen "
+"Registrations-Phantasien welche gewisse Regierungen, entgegen zu wirken.\n"
+"    <p>Hier eine kleine Übersicht was wir am bauen sind:\n"
+"    "
 
 #: modelreg/templates/about.html:11
 msgid ""
 "A web application where people can register themselves (voluntarily, address "
 "verification via physical letter)"
 msgstr ""
+"Eine Web-Applikation auf welcher sich die Leute selber registrieren können "
+"(freiwillig, Adressen-Verifizierung via Physikalischem Brief)"
 
 #: modelreg/templates/about.html:12
 msgid ""
 "Ability to print out a sheet with QR codes to stick to your model "
 "(potentially as-a-service for a small fee)"
 msgstr ""
+"Die Möglichkeit QR-Code Klebeetiketten auszudrucken um am Modell anzubringen "
+"(Möglich als Dienstleistung gegen eine kleine Gebühr)"
 
 #: modelreg/templates/about.html:13
 msgid ""
 "The data will be fully encrypted. Part of the QR code URL would be a random "
 "key to decrypt part of your information (for example phone number and name)"
 msgstr ""
+"Alle Daten sind voll verschlüsselt. Ein Teil des QR-Codes ist ein zufällig "
+"generierter Schlüssel, welcher zum entschlüsseln der Daten benötigt wird "
+"(wie zum Beispiel die Telefonnummer und der Name)"
```

Comment:
Unrelated to the translation - Maybe we should fix this text as long as this is not yet implemented..

Reply:
Yes, I thought about that too. But to get the translation going, I just translated the English text.

Reply:
Let's fix that in a separate issue/PR then.

Reply:
See #43